code-samples/eventing/bookstore-sample-app/frontend/client/components/SubmitButton.js

```diff
@@ -0,0 +1,18 @@
+'use-client';
+import { useState } from 'react';
+
+export default function SubmitButton() {
+	const [hover, setHover] = useState(false);
+
```

Comment:
Could you leave a placeholder function here? Like after click the submit button, we get a javascript alert or console log. It will be convenient up implement the form submission action as next step!